### ISSUE TITLE
Store availability?

### ISSUE THREAD

**Issue by andrewducker** [LuceCarter/EmotiPal#1]
Hi! Is this available on the store? I checked Google Play and couldn’t find it.

Would love to make this available to friends, but they’d need to have a really simple install method.

**Comment by LuceCarter** [comment]
Hi Andrew,

Its not at the moment as I wasn't sure if two features was enough.

Plus I would want to make sure it passes A11Y front-end checks such as colour contrast etc.


Its on the radar though! I could perhaps release Android first soon though? That has fewer hoops to jump through than Apple to get onto the store.

**Comment by andrewducker** [comment]
When the two features are both so big and useful I think neurodivergent people would find it handy.  And I'm a big fan of the "release early, release often" ethos.

But obviously it's your application, and I don't want you to feel pressure from a complete stranger!

**Comment by LuceCarter** [comment]
I agree about release early, release often. I can release a beta version to the store.

I cannot be 100% sure it is bug free so wouldn't want to call it complete.

Would rather get it out there to help people and then work on any issues when they come up!

I have wanted to get an app into the store for a while, just never felt I had one ready.

This might need some tweaks first as a couple of responses are for demo's so have jokes in I can explain. So I will remove these and be more literal in the answers.

But thank you for the encouragement!

**Comment by andrewducker** [comment]
I shall look forward to it.  And thank you for taking it onboard.
